platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PreconsensusEventMetrics.java

```diff
@@ -63,8 +63,9 @@ public class PreconsensusEventMetrics {
     private static final RunningAverageMetric.Config PRECONSENSUS_EVENT_AVERAGE_UN_UTILIZED_FILE_SPAN_CONFIG =
             new RunningAverageMetric.Config(CATEGORY, "preconsensusEventAverageUnutilizedFileSpan")
                     .withUnit("generations")
-                    .withDescription("The average unutilized generational span of preconsensus event files. "
-                            + "Only reflects files written since the last restart. Smaller is better.");
+                    .withDescription(
+                            "The average unutilized generational span of preconsensus event files prior "
```

Comment:
This amendment "prior to span compaction" should also be added to the metric config above, `PRECONSENSUS_EVENT_AVERAGE_FILE_SPAN_CONFIG`

Reply:
Not actually necessary. The metric above reflects the actual generational span in the file, this metric shows the different between the actual span and the span specified by the fields in the file name. After span compaction is complete, the span specified by file name will match the actual span of events in the file.

The reason why it's still important to report this metric is that if we crash prior to finishing a file we will never get the chance to do span compaction. This is not the end of the world, but it may cause a large overlap between files, which is undesirable for performance reasons (it could cause us to take longer to replay the files).